src/notifications/api/tests/test_abonnement.py

```diff
@@ -156,3 +156,35 @@ def test_abonnement_update_kanalen(self):
 
         self.assertEqual(len(kanalen), 1)
         self.assertEqual(kanalen.pop().naam, 'zaken')
+
+    def test_abonnementen_create_inconsistent_filters(self):
+        """
+        test /abonnementen POST:
+        create abonnement with filters inconsistent with kanaal filters
+        check if response contains status 400
+        """
+
+        KanaalFactory.create(naam='zaken')
+        abonnement_create_url = get_operation_url('abonnement_create')
+
+        data = {
+            "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+            "auth": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImNsaWVudF9pZG"
+                    "VudGlmaWVyIjoienJjIn0.eyJpc3MiOiJ6cmMiLCJpYXQiOjE1NTI5OTM"
+                    "4MjcsInpkcyI6eyJzY29wZXMiOlsiemRzLnNjb3Blcy56YWtlbi5hYW5t"
+                    "YWtlbiJdLCJ6YWFrdHlwZXMiOlsiaHR0cDovL3p0Yy5ubC9hcGkvdjEve"
+                    "mFha3R5cGUvMTIzNCJdfX0.NHcWwoRYMuZ5IoUAWUs2lZFxLVLGhIDnU_"
+                    "LWTjyGCD4",
+            "kanalen": [{
+                "naam": "zaken",
+                "filters": [
+                    {"bron": "082096752011"},
+                    {"zaaktype": "example.com/api/v1/zaaktypen/5aa5c"},
+                    {"vertrouwelijkheidaanduiding": "*"}
+                ]
+            }]
+        }
+
+        response = self.client.post(abonnement_create_url, data)
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.data)
```

Comment:
I want to see more assertion about the type of error that happened. You can look at `zrc.api.tests.test_validation` in ZRC for some inspiration

Reply:
Assertion of the type of the error are added